src/mixins/_mixins.scss

```diff
@@ -1,3 +1,4 @@
 @import 'stack/stack';
 @import 'font-smoothing/font-smoothing';
 @import 'global-styles/global-styles';
+@import 'components/input-styles';
```

Comment:
This is weird. You can use `yarn scaffold:mixin input` to generate a mixin in the mixins dir. Rather than importing it from the component.